pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook.go

```diff
@@ -33,13 +33,19 @@ type Webhook struct { // nolint:maligned
 	file.BoilerplateMixin
 	file.ResourceMixin
 
-	// Is the Group domain for the Resource replacing '.' with '-'
-	GroupDomainWithDash string
-
+	// Version of webhook marker to scaffold
+	WebhookVersion string
```

Comment:
```suggestion
	ApiVersion string
```

Since you would like to have the arg as `webhook-version` instead `api-version` then, in POV we can have the controller/model layers following the k8s nomenclature.

Reply:
How do you suggest we introduce `webhookVersions={v1beta1},` to the webhook marker then, if not by templating?

Reply:
Could not to be something such as : 

```
// +kubebuilder:webhook:webhookVersions={{ f.ApiVerision}}
```

Are you adding both `{v1beta1, v1}` when the apiVersion used is the `v1beta1`? Could we update the generate shell script for the v3 project have this scenario for we are able to check and validate it?